addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,55 @@
+# iCloud Binding
+
+The iCloud Binding helps to integrate apple device information into openHAB.
```

Comment:
The first paragraph of a binding readme will be extracted for http://docs.openhab.org/addons/bindings
Feel free to add a few more details in a second sentence (like "retrieve location and battery details" or "through the publicly available iCloud")

Reply:
I´ve rephrased the first sentence; I´ll commit later today.